components/service/fretboard/build.gradle

```diff
@@ -25,6 +25,8 @@ dependencies {
     implementation project(':support-ktx')
     implementation "org.jetbrains.kotlin:kotlin-stdlib:${rootProject.ext.dependencies['kotlin']}"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:${rootProject.ext.dependencies['coroutines']}"
+    implementation "android.arch.work:work-runtime:${rootProject.ext.dependencies['workmanager']}"
+    implementation "android.arch.work:work-firebase:${rootProject.ext.dependencies['workmanager']}"
```

Comment:
Is that needed?

If yes: Can you comment on the issue about that? And can you run the androidDependencies task and see what else this pulls in? Especially I'm interested in knowing whether this requires closed-source Firebase dependencies.